### PR TITLE
Add ArangoDBStorage

### PR DESCRIPTION
**Pull request by joakimnordling**
The storage can be used like this:
```
from arango import ArangoClient
from migrate_anything import configure, ArangoDBStorage

client = ArangoClient(hosts="https://example.com:8529/")
db = client.db(name="my_db", username="root", password="supersecret")

configure(storage=ArangoDBStorage(collection="migrations", db=db))
```

### CODE REVIEW

**Comment by lietu** [comment]
Please provide example in the examples folder 🙂

**Comment by lietu** [comment]
Will have to think about how to document all the different storage classes in clear manner, but don't need to solve it with this PR

**Comment by joakimnordling** [comment]
Now there's an example in the examples folder.

File: examples/arangodb/migrations/01-test.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from arango import ArangoClient
+
```
Comment (lietu) [inline examples/arangodb/migrations/01-test.py:2]: Maybe add
```
# Don't actually put your credentials in these files as they get stored in the DB in readable format then
```

Comment (joakimnordling) [reply examples/arangodb/migrations/01-test.py:2]: Added.

**Review by lietu — APPROVED** [review]

**Pull request merged**